force-app/main/default/lwc/todoList/__tests__/todoList.test.js

```diff
@@ -0,0 +1,108 @@
+        expect(lightningInputEl.checked).toBeFalsy();
+
+        // Query for rendered list items
+        const listItemEls = element.shadowRoot.querySelectorAll('li');
+        expect(listItemEls.length).toBe(0);
+
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise ends in the
+        // rejected state
+        return Promise.resolve().then(() => {
+            // Compare if tracked property has been assigned a new value.
+        });
+    });
+
+    it('renders multiple list items and filters based on priority', () => {
+        // Create initial element
+        const element = createElement('c-todo-list', {
+            is: TodoList
+        });
+        element.todos = todos;
+        document.body.appendChild(element);
+
+        // Query list items for initial values
+        let listItemEls = element.shadowRoot.querySelectorAll('li');
+        expect(listItemEls.length).toBe(2);
+
+        todos.push({
+            id: 3,
+            description: 'Test todo list',
+            priority: false
+        });
+
+        element.todos = todos;
+
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise ends in the
+        // rejected state
+        return Promise.resolve()
+            .then(() => {
+                // Check if rendered list got updated with three items
+                listItemEls = element.shadowRoot.querySelectorAll('li');
+                expect(listItemEls.length).toBe(3);
+
+                const lightningInputEl = element.shadowRoot.querySelector(
+                    'lightning-input'
+                );
+                lightningInputEl.checked = true;
+                lightningInputEl.dispatchEvent(new CustomEvent('change'));
+            })
+            .then(() => {
+                // Check if rendered list got filtered based on priority flag
+                listItemEls = element.shadowRoot.querySelectorAll('li');
+                expect(listItemEls.length).toBe(1);
+            });
+    });
+
+    it('renders and filters list items', () => {
+        // Create initial element
+        const element = createElement('c-todo-list', {
+            is: TodoList
+        });
+        element.todos = todos;
+        document.body.appendChild(element);
+
+        // Return a promise to wait for any asynchronous DOM updates. Jest
+        // will automatically wait for the Promise chain to complete before
+        // ending the test and fail the test if the promise ends in the
+        // rejected state
+        return Promise.resolve().then(() => {
+            // Validate rendered output for first todo object
+            let outputEls = element.shadowRoot.querySelector('p');
```

Comment:
`querySelectorAll`? Did you run this locally?

Or instead of querying for the `p` elements, you could get the `textContent` of the `li` that contains the todo and validate the full text (via `RegExp` if that's easier). Then you have a looser dependency on the DOM structure and can potentially catch issues with extra junk being in the todo output.